infra/datalake/processing_stack.py

```diff
@@ -107,9 +108,9 @@ def __init__(self, scope: core.Construct, stack_id: str, deploy_env, vpc, **kwar
             self,
             "batch-launch-template",
             launch_template_name="datalake-batch-launch-template",
-            launch_template_data={
```

Comment:
This change looks OK, but from experience I know that it might fail. We are missing infrastructure tests to make sure these kind of changes are not breaking things. To include this change now you basically need to deploy all stacks and try to launch one AWS Batch job for testing.

Reply:
Otherwise this PR looks good to me.

Reply:
Deployed arn:aws:cloudformation:ap-southeast-2:632223577832:stack/geospatial-data-lake-processing-dev/aac81e40-3b56-11eb-826d-02136ea02430 successfully. Tested `datasets-endpoint` with this input:

```
{
  "httpMethod": "POST",
  "body": {
    "type": "RASTER",
    "title": "test dataset",
    "owning_group": "A_ABC_XYZ"
  }
}
```

Result:

```
{
  "statusCode": 201,
  "body": {
    "created_at": "2020-12-11T02:24:42.965803+0000",
    "id": "073eed563b5811eb8c8ebabcf0bb2325",
    "owning_group": "A_ABC_XYZ",
    "title": "test dataset",
    "type": "RASTER",
    "updated_at": "2020-12-11T02:25:22.297354+0000"
  }
}
```

Reply:
@l0b0 , sorry, but `datasets-endpoint` has nothing to do with AWS Batch Launch template. AWS Batch Launch template is used in State Machinge workflow which is part of `datasets-versions-endpoint`. To test AWS Batch Launch template you basically need to submit AWS Batch job and see if Compute Evironment (ECS instances are launched) is successfully created.

Reply:
I've addressed this by paring back this PR to something less ambitious.